src/synced_services/redmine/redmine_synced_service.ts

```diff
@@ -472,6 +476,23 @@ export class RedmineSyncedService implements SyncedService {
     );
 
     if (!response || !response.ok) {
+        if (response.status === 422) {
+          const error = new Error();
+
+          error.service = "Redmine";
+          error.exception = response.body.errors;
+        //console.error(res.body.errors);
+          let context = null;
+          if (timeEntryBody) {
+          //console.error(body);
+           context = new ExtraContext()
+          context.name = "Time entry";
+          context.context = JSON.parse(JSON.stringify(timeEntryBody))
+          error.data = JSON.parse(JSON.stringify(timeEntryBody));
```

Comment:
Rad by som toto cele riesil vo vlastnom service. Nieco ako ErrorService, s tym ze volam nieco ako error  = errorService.getRedmineError(response.body.errors, timeEntryBody), context = errorService.getRedmineTimeEntryCOntext(...).
Nevadi, ze bude tato funkcia pouzita len raz / 2 krat v ramci celeho kodu, ide o to, aby zbytocne nespinila tento kod, ktory je uz ajtak dost neprehladny. 

Pripadne to kludne mozes este rozdelit na ErrorService a ContextService, kde kazdy bdue riesit vlastnu vec.

Reply:
Zaroven by ten ContextService mohol zrejme byt pouzivany iba priamo v SentryService, nakolko outside of sentryService funkcii som nenasiel pouzitie contextu (ak je, daj mi vediet).

Reply:
Melo by byt upraveno.